### PR TITLE
Add collections in EE image

### PR DESCRIPTION
**Pull request by Akasurde**
* amazon.aws
* vmware.vmware_rest

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

**Comment by Akasurde** [comment]
cc @goneri

File: .zuul.d/project-templates.yaml [hunk 1]
```diff
@@ -24,6 +25,7 @@
               - name: github.com/ansible-collections/servicenow.itsm
               - name: github.com/ansible-collections/splunk.es
               - name: github.com/ansible-collections/trendmicro.deepsec
+              - name: github.com/ansible-collections/vmware.vmware_rest
```
Comment (goneri) [inline .zuul.d/project-templates.yaml:28]: We also need cloud.common.

**Review by goneri — CHANGES_REQUESTED** [review]

**Comment by goneri** [comment]
Thank you @Akasurde for doing this.

File: .zuul.d/project-templates.yaml (hunk 1 above)
Comment (Akasurde) [reply .zuul.d/project-templates.yaml:28]: Cloud.common is already part of this file, added via https://github.com/ansible/network-ee/pull/65.

**Pull request merged**